movie/src/components/Card/index.jsx

```diff
@@ -33,6 +40,10 @@ const Card = ({ movies }) => {
     setCardId(0);
   };
 
+  // use on style.
+  const isCircleColor = (voteAverage) =>
```

Comment:
`isCircleColor` like a value that return `true`||`false`. So please find a better name for this